### PR TITLE
Suppression rules

### PR DESCRIPTION
**Pull request by paulmendix**
Add documentation for the new delighter that makes it possible to suppress warnings that you see in Studio Pro.

The delighter will be released in Mx 8.7

### CODE REVIEW

**Comment by MariaShaposhnikova** [comment]
@mrseanryan , can you please sign CLA for the PR to pass the checks? Thanks.

**Comment by paulmendix** [comment]
We expect more changes to the documentation because we improved the feature during the crafting days.

File: content/refguide/suppression-rules.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+After you select the menu, a confirmation appears:
+
+![Confirmation that a warning was suppressed](attachments/suppression-rules/confirmation-dialog-after-suppressing-a-warning.png)
+
+The list of warnings will be updated in the error pane.
+
+To modify or undo the suppression of the warnings, see section [Editing the suppression rules](#editing-rules).
+
+### 2.4 Suppress App Store warnings
```
Comment (dlhartveld) [inline content/refguide/suppression-rules.md]: Add anchor for app store:

```suggestion
### 2.4 Suppress App Store warnings {#suppress-appstore-warnings}
```

**Comment by MariaShaposhnikova** [comment]
@paulmendix , @mrseanryan , thank you for writing this document! I have restructured it and aligned with our style guide.

**Pull request merged**